backends/events/sdl/sdl-events.cpp

```diff
@@ -77,6 +77,7 @@ SdlEventSource::SdlEventSource()
 	  , _queuedFakeKeyUp(false), _fakeKeyUp(), _controller(nullptr)
 #endif
 	  {
+#ifndef KOLIBRIOS
```

Comment:
As discussed, it would be best to implement those missing SDL functions, so these `#ifdefs` could be safely dropped

Reply:
Done